src/settings.ts

```diff
@@ -1,5 +1,8 @@
 import BurnerConnectProvider from "@burner-wallet/burner-connect-provider";
 
+// hide some data to send less queries
+export const ETHDENVER_OPTIMIZATION = true;
```

Comment:
makes this true for every platform?  should this be just for when the NETWORK is xdai?